website/docs/events/multiplier-event-1.md

```diff
@@ -0,0 +1,39 @@
+# Multiplier Event 1: Pre-launch of 1st content
+
+- Date: 27.10.2022
+- Time: 9:00 - 14:00 EEST
+- Place: Room PR 105, Faculty of Automatic Control and Computer Science, POLITEHNICA University of Bucharest
+
+## Initial Presentation: 9:00 - 11:30
```

Comment:
```suggestion
## Initial Presentation: 9:00 - 11:00
```

Reply:
You are highly optimistic that we can talk for only ~24 minutes each, questions included. You in particular :)))